config/manifests/bases/opendatahub-operator.clusterserviceversion.yaml

```diff
@@ -6,7 +6,7 @@ metadata:
     capabilities: Full Lifecycle
     categories: AI/Machine Learning, Big Data
     certified: "False"
-    containerImage: quay.io/opendatahub/opendatahub-operator:v2.33.0
+    containerImage: quay.io/opendatahub/opendatahub-operator:v3.0.0
```

Comment:
Was this base file supposed to be updated in your uplift PR @zdtsw ? or should this be left as is.

Reply:
i think 2.33.0 is def a wrong value.
if we are heading tp 3.0.0 on main, it _should_ be correct
as 2.35.0 will be out from stable-2.x